### PR TITLE
api-lifecycle-updated

### PR DESCRIPTION
**Pull request by roopksaini**
Screenshot 2024-07-17 at 11 58 47 AM

### CODE REVIEW

File: apis/overview/lifecycle.md [hunk 1]
```diff
@@ -27,33 +27,62 @@ The following changes are backward compatible:
...
-API product releases have a well-defined lifecycle. The release status determines the level of support as described below.
+API product releases have a well-defined lifecycle. The release status determines the level of support we offer for the API.
+
+![api-lifecycle-diagram](/images/API-Lifecycle-complete.jpg)
+
+### Tech Preview
+
+We may release new APIs, features, and operations as a Tech Preview. This allows you to review upcoming capabilities and provide feedback. Additionally, it assists our team in making sure we are providing the right solutions and expected functionality.
+
+You may use APIs in Tech Preview for free while they are in preview. Do not use these APIs in Production. They are available for testing only, to gather feedback before releasing for General Availability.
```
Comment (irenajur) [inline apis/overview/lifecycle.md:42]: Is this applicable for tech-preview versions of APIs that already have GA version?

File: apis/overview/lifecycle.md [hunk 2]
```diff
@@ -27,33 +27,62 @@ The following changes are backward compatible:
...
+| --------------------- | -------------------------------- | ------------------------------------------ | --------------------------------- |
+| **Limited Support**   | **Full Support**                 | **Limited Support**                        | **Upgrade Only**                  |
+| GitHub community      | Bentley support                  | Bentley support (for one year)             | Upgrade to the latest API version |
+| Stack Overflow        | GitHub community                 | Github community                           |                                   |
+| Security updates      | Stack Overflow                   | Security updates                           |                                   |
+| New functionality     | Security updates                 | Maintenance updates (Bentley's discretion) |                                   |
+| Bug fixes             | Maintenance updates              | No enhancements or new functionality       |                                   |
+|                       | New functionality - Non breaking |                                            |                                   |
+
+<!-- ## Deprecation Guidelines
```
Comment (irenajur) [inline apis/overview/lifecycle.md:72]: why this part is commented out?

File: apis/overview/lifecycle.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 ## API versions
 
-All APIs are versioned and the version number is embedded in the **Accept** header. Sample request:
+Our team versions all APIs and embeds the version number in the **Accept** header as shown below.
 
-``` HTTP
+```HTTP
 GET https://api.bentley.com/users/me HTTP/1.1
 Authorization: Bearer JWT_TOKEN
 Accept: application/vnd.bentley.itwin-platform.v1+json
```
Comment (irenajur) [inline apis/overview/lifecycle.md:14]: No need to keep extra line at the end of code block

File: apis/overview/lifecycle.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
 <!-- Copyright (c) Bentley Systems, Incorporated. All rights reserved.            -->
 <!-- See LICENSE in the project root for license terms and full copyright notice. -->
 
-# API release lifecycle
+# API versioning and release lifecycle
 
 ## API versions
 
-All APIs are versioned and the version number is embedded in the **Accept** header. Sample request:
+Our team versions all APIs and embeds the version number in the **Accept** header as shown below.
```
Comment (irenajur) [inline apis/overview/lifecycle.md:8]: Maybe it's a matter of taste, but do we need to focus on "our team" here? The main point is to highlight the versioning, not what our team does

Comment (roopksaini) [reply apis/overview/lifecycle.md:8]: I feel what you are saying. I was trying to follow the style guide spot on. It says to use active voice vs. passive voice.

File: apis/overview/lifecycle.md (hunk 2 above)
Comment (roopksaini) [reply apis/overview/lifecycle.md:72]: Michelle said to leave it out for now. But we might use it in a future revision

File: apis/overview/lifecycle.md (hunk 1 above)
Comment (roopksaini) [reply apis/overview/lifecycle.md:42]: Not sure. Will have to check

Comment (roopksaini) [reply apis/overview/lifecycle.md:42]: Yes, this is applicable for tech-preview versions of APIs with a GA version. The tech-preview version - which will be free - is only meant to be used for testing. Not in production.

File: apis/overview/lifecycle.md (hunk 4 above)
Comment (roopksaini) [reply apis/overview/lifecycle.md]: Changed to we/us instead of "our team" throughout.

File: apis/overview/lifecycle.md (hunk 2 above)
Comment (irenajur) [reply apis/overview/lifecycle.md:72]: Should we still keep it as commented out in public repo?

Comment (roopksaini) [reply apis/overview/lifecycle.md]: Removed it from this branch. Moved to a [separate branch](https://github.com/iTwin/platform-api-documentation/blob/api-lifecycle-future-update/apis/overview/lifecycle.md?plain=1#L72) for future update.

**Review by irenajur — APPROVED** [review]
lgtm

**Pull request merged**